### PR TITLE
Add simplePDF

### PR DESCRIPTION
**Pull request by bendersej**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -939,6 +939,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Wufoo](https://www.wufoo.com/) - Quick forms to use on websites. Free plan has a limit of 100 submissions each month.
   * [formpost.app](https://formpost.app) - Free, unlimited Form to Email service. Setup custom redirect, auto response, webhooks etc for free.
   * [Formester.com](https://formester.com) - Share and embed unique-looking forms on your website—no limits on the number of forms created or features restricted by the plan. Get up to 100 submissions every month for free.
+  * [SimplePDF.eu](https://simplepdf.eu/embed) - Embed a PDF editor on your website and turn any PDF into a fillable form. Free plan allows unlimited PDFs with 3 submissions per PDF.
```
Comment (ripienaar) [inline README.md:942]: Does it mean any pdf on your site can only be edited twice? As in this free service works only twice then you have to make a new document?

Comment (bendersej) [reply README.md:942]: The PDFs can be edited and filled as many times as possible (and downloaded locally). Viewing the resulting submissions by the owner of the website (who embedded the PDF editor) is limited to 3 per document.

You can see an example in action [here](https://schrutefarms.bendersej.repl.co)

Happy to rephrase any part of the explanation!

Comment (ripienaar) [reply README.md:942]: That’s ok - well maybe clarify on the website but this is fine for here

**Comment by ripienaar** [comment]
Thank you

**Comment by bendersej** [comment]
Thanks you for maintaining this list! 

Have a great end of weekend!

**Pull request merged**